### DISCUSSION TITLE
Adding a general bookmark from pressing a key on hands free

### DISCUSSION THREAD

**Discussion by Archie-2021** [PaulWoitaschek/Voice#2277] [Ideas]
It is supper useful UX to just double tabbing on pause button or something like that and that add a bookmark to add notes on that timestamp later. I am not sure it is a OS setting or relates to app itself , but I have seen such a feature in other appa.

**Comment by ZoranLain** [comment]
I found that you can long press the bookmark icon at the top of the playback screen to quickly add a bookmark without having to go into the bookmark menu.  Very handy.
